packages/terra-clinical-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -64,6 +64,8 @@ const propTypes = {
   selectableRefCallback: PropTypes.func,
 };
 
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
```

Comment:
can we remove these?

Reply:
I was able to remove the `eslint-disable` for the tabIndex. But for `jsx-a11y/no-static-element-interactions` linter throws up if we don't provide any specific role.